package.json

```diff
@@ -71,7 +70,6 @@
 		"test:rule-help-version": "mocha test/test-rule-help-version.js",
 		"test:node": "mocha test/node/*.js",
 		"version": "echo \"use 'npm run release' to bump axe-core version\" && exit 1",
-		"prepublishOnly": "grunt build && grunt file-exists",
```

Comment:
We should still have a `prepublishOnly` (or `prepare`) script. Otherwise we'll have to manually run the build before we `npm publish`. Having this here reduces the chances of publishing a broken version of axe.

Reply:
the publish script already runs the `verify-release` script which checks if all files are there, so I figured we didn't need this one anymore https://github.com/dequelabs/axe-core/blob/develop/.circleci/config.yml#L176

Reply:
I see. I suppose we can safely remove this as long as we always remember to do `npm run build` first. 🤷